packages/web3-eth/src/rpc_method_wrappers.ts

```diff
@@ -1926,22 +1922,6 @@ export async function getFeeHistory<ReturnFormat extends DataFormat>(
 
 	return format(feeHistorySchema, response as unknown as FeeHistory, returnFormat);
 }
-/**
- *
- * @param web3Context
- * @param transaction
- */
-async function getRevertReason<ReturnFormat extends DataFormat>(
```

Comment:
Method was moved to `utils/get_revert_reason.ts` because it's not a RPC method

Reply:
get revert reason is wrapper on `eth_call` rpc method, so it was under `rpc_method_wrappers.ts`,

Reply:
What I meant is that `getRevertReason` isn't a wrapper of it's own RPC method such as `eth_getRevertReason`. Other reasons I removed it from `rpc_method_wrappers.ts` is because it's no longer exported to the user (so that we can make changes to it as needed without it being a breaking change) and the function is a utility method to be used by other methods, not on it's own